site/hugo.toml

```
@@ -208,7 +208,7 @@ enable = true
 
 [menu]
   [[menu.main]]
-    name = "Contributions"
+    name = "Community"
     weight = -98
     url = "/contributions"
```

```suggestion
    url = "/community"
```
@envoyproxy/gateway-maintainers will this require renaming the `contributions` directory to `community`?

---

@Soham-47 don't think the URL rename will work without changing the directory name. Let's wait for opinions from others if we really want to do that.

---

ok sure